packages/react/src/oxygen-ui-react-auth-components/SignInButton/sign-in-button.scss

```diff
@@ -20,7 +20,7 @@
   margin: 16px 0;
 }
 
-.OxygenSignInButtonSocial {
+.OxygenSignInButton-social {
```

Comment:
Is `MUI` using `--` for variations?

```suggestion
.OxygenSignInButton--social {
```

Reply:
`MUI` uses `-` for variations. 
Ex: They use `MuiButton-contained` class name for `contained` variation. Refer [Mui Button.](https://mui.com/material-ui/react-button/#basic-button)

Reply:
Ack 🙌